tests/workflows/fastq_download_prefetch_fasterqdump_sratools/main.nf.test

```diff
@@ -32,6 +32,42 @@ nextflow_workflow {
                 { assert workflow.success },
                 { assert snapshot(workflow.out).match() }
             )
+
+            assert path("$launchDir/output").exists()
+
+            def lines = path("$launchDir/output/prefetch/failures_report.csv").readLines()
+            assert lines.contains("test_single_end,DRR000774").equals(false)
+            assert lines.contains("test_paired_end,SRR11140744").equals(false)
+        }
+    }
+
+    test("Download errors: 403") {
+
+        when {
+            workflow {
+                """
+                input[0] = Channel.of(
+                    [[ id:'SAMPLE1', single_end:false ], 'ERR1109373'],
+                    [[ id:'ERROR1', single_end:false ], 'SRR999908'],
+                    [[ id:'ERROR2', single_end:false ], 'SRR999934'],
```

Comment:
Are both these errors for reads that have access denied? Could one of the errors be for an accession that doesn't exist or is invalid?

Reply:
Yeah, both 403 errors.

Reply:
Updated in https://github.com/phac-nml/fetchdatairidanext/commit/bbf2e3fe013b902b060af3ca063f5fad25f837fd